integration-test/src/main/java/com/sequenceiq/it/cloudbreak/mock/DefaultModel.java

```diff
@@ -122,4 +136,26 @@ public void terminateInstance(Map<String, CloudVmMetaDataStatus> instanceMap, St
         CloudVmMetaDataStatus cloudVmMetaDataStatus = new CloudVmMetaDataStatus(cloudVmInstanceStatus, vmMetaDataStatus.getMetaData());
         instanceMap.put(instanceId, cloudVmMetaDataStatus);
     }
+
+    public void stopAllInstances() {
```

Comment:
maybe the stopAllInstances and startAllInstances methods could call a more generic third - private - method, since their bodies are almost the same